cpp/src/gandiva/projector.h

```diff
@@ -138,6 +167,7 @@ class GANDIVA_EXPORT Projector {
   SchemaPtr schema_;
   FieldVector output_fields_;
   std::shared_ptr<Configuration> configuration_;
+  bool compiled_from_cache_;
```

Comment:
nit: use a different name like built_from_cache since it is already compiled

Reply:
built_from_cache sounds better, so changed to it.